contracts/system_addresses.go

```diff
@@ -19,6 +19,8 @@ var (
 	ChildERC20Contract = types.StringToAddress("0x1003")
 	// ChildERC20PredicateContract is an address of child ERC20 predicate contract on the child chain
 	ChildERC20PredicateContract = types.StringToAddress("0x1004")
+	// EIP1559BurnContract is an address of EIP-1559 burn contract on the child chain
+	EIP1559BurnContract = types.StringToAddress("0x104")
```

Comment:
Nitpick:
move it below `MerkleContract` (https://github.com/0xPolygon/polygon-edge/pull/1323/files#diff-cb0d9c0c8a217bb409554571a2a979efd58f1dbf13e059824db4b1df98abed3cR11), so that the addresses are ordered by values ascending.